integration/key/release_name.go

```diff
@@ -0,0 +1,29 @@
+package key
+
+import (
+	"fmt"
+)
+
+func AWSConfigReleaseName(clusterID string) string {
```

Comment:
Shouldn't this be for Azure?

Reply:
I copied the whole file from the `aws-operator`. Dropped the dead code.